common/src/test/java/io/netty/util/concurrent/FastThreadLocalTest.java

```diff
@@ -76,6 +80,30 @@ public void run() {
         }
     }
 
+    @Test
+    public void testMultipleSetRemove() throws Exception {
+        final FastThreadLocal<String> threadLocal = new FastThreadLocal<String>();
+
+        final Field liveSetField = ObjectCleaner.class.getDeclaredField("LIVE_SET");
+        liveSetField.setAccessible(true);
+        final Set<?> liveSet = (Set<?>) liveSetField.get(ObjectCleaner.class);
+        final int sizeWhenStart = liveSet.size();
+
+        Thread thread = new Thread(new Runnable() {
+            @Override
+            public void run() {
+                threadLocal.set("1");
+                threadLocal.remove();
+                threadLocal.set("2");
+                threadLocal.remove();
+            }
+        });
```

Comment:
Also can you please use at least two threads to ensure everything is fine even when we have more then one ?